source/faq.txt

```diff
@@ -4,12 +4,153 @@ FAQ
 
 .. default-domain:: mongodb
 
-Frequently Asked Questions
-
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-Lorem ipsum.
+General
+-------
+
+Why are there two types of ``MongoClient`` in the Java driver? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two types of ``MongoClient`` because we wanted a cleaner API
+for new users that didn't have the confusion of including multiple CRUD
+API's. We wanted to ensure that the new CRUD API was available in a Java
+package structure that would work well with Java module support
+introduced in Java 9. 
+
+Which type of ``MongoClient`` should I use?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should use ``com.mongodb.MongoClient`` class if you want the
```

Comment:
I would like to emphasize/encourage use of the new API, so consider re-organizing to something like this:

```
New applications should generally use the ``com.mongodb.client.MongoClient`` interface. It supports:

- Configuration with ``MongoClientSettings`` and ``ConnectionString``.  You can create instances of this interface via factory methods defined in the ``com.mongodb.client.MongoClients`` class. 
- CRUD API using ``MongoDatabase``, and from there, ``MongoCollection``

You should use ``com.mongodb.MongoClient`` class if you require support for the legacy API, which includes:

- Configuration with ``MongoClientOptions`` and ``MongoClientURI``
- CRUD API using ``DB``, and from there, ``DBCollection``.  This API is accessed via the ``getDB`` method.

For applications that require a mix of the new and legacy APIs, ``com.mongodb.MongoClient`` also supports: 

- Configuration with ``MongoClientSettings`` and ``ConnectionString``, the only difference being that instances are created via constructors instead of a factory class.
- CRUD API using ``MongoDatabase``, and from there, ``MongoCollection``.  This API is accessed via the ``getDatabase`` method.
```